File: 217-ContainsDuplicate.go
func containsDuplicate(nums []int) bool {
    res := make(map[int]int)
    for i:= range(len(nums)) {
        if _,ok := res[nums[i]]; ok{
            res[nums[i]] += 1
        } else {
            res[nums[i]] += 1
        }
        if res[nums[i]] >= 2 {
            return true
        }
    }    
    return false
}

File: 217-ContainsDuplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        ele_count = {}
        for i in nums:
            if i in ele_count:
                ele_count[i] += 1
            else:
                ele_count[i] = 1
            if ele_count[i] >= 2:
                return True
        return False
        